src/test/java/org/folio/consortia/controller/UserTenantControllerTest.java

```diff
@@ -128,7 +167,13 @@ void return404UserTenantNotFoundWithUserId() throws Exception {
   @Test
   void getValidationError() throws Exception {
     var headers = defaultHeaders();
-    this.mockMvc.perform(get("/consortia/user-tenants?userId=90unnn").headers(headers))
+    ConsortiumEntity consortiumEntity = new ConsortiumEntity();
+    consortiumEntity.setId(UUID.fromString("7698e46-c3e3-11ed-afa1-0242ac120002"));
+    consortiumEntity.setName("TestConsortium");
```

Comment:
please extract separate method to create ConsortiumEntity

Reply:
fixed.